torch/onnx/symbolic_opset10.py

```diff
@@ -178,3 +178,13 @@ def flip(g, input, dims):
 
 def fmod(g, input, other):
     return g.op("Mod", input, other, fmod_i=1)
+
+@parse_args('v', 't', 'i', 'i', 'i')
+def fake_quantize_per_tensor_affine(g, inputs, scale, zero_point, quant_min=-128, quant_max=127):
+    if quant_min not in [0, -128] or quant_max not in [127, 255]:
+        raise ValueError("ONNX defines [0, 255] for quint8 and [-128, 127] for qint8, got [{}, {}]".format(
```

Comment:
Thanks for adding the symbolic. Could you please use _unimplemented() function in this case to keep the behavior consistent across symbolics?

Reply:
Updated to use _unimplemented()